app/index.js

```diff
@@ -89,7 +88,7 @@ module.exports = class extends Generator {
 				name: this.user.git.name(),
 				email: this.user.git.email(),
 				website: props.website,
-				humanizedWebsite: humanizeUrl(props.website),
+				normalizedWebsite: props.website,
```

Comment:
This is just the same as the `website` item.

Reply:
@sindresorhus  Right. fixed it in the lastest commit :)